log.conf

```diff
@@ -1,3 +1,3 @@
 path = /var/log/multi-cloud
-level = debug
```

Comment:
it's better put it into multi-cloud/example/multi-cloud.conf

Reply:
It seems there is no multi-cloud/example/multi-cloud.conf.